test/Microsoft.ApplicationInsights.AspNetCore.Tests/project.json

```diff
@@ -15,19 +15,22 @@
     "Microsoft.Extensions.Logging": "1.0.0-rc2-*",
     "Microsoft.Extensions.Configuration.Json": "1.0.0-rc2-*",
     "xunit": "2.1.0",
-    "FunctionalTestUtils": "1.0.0-rc2-*"
+    "FunctionalTestUtils": "1.0.0-rc2"
```

Comment:
Removed it and replaced it with "*". Also removed AI.AspNetCore dependency which is added in addition to the project reference that we get from FunctionalTestUtils. Thanks!